templates/types/streaming/fastapi/app/api/controllers/file.py

```diff
@@ -14,11 +15,40 @@
 from app.engine.index import get_index
 
 
+class RawFileReader(BaseReader):
```

Comment:
oops, no reader for txt files in llama_index?

Reply:
I checked the default readesr from the SimpleDirectoryReader but it doesn't support the txt files. But after looking around the source code, there is a FlatReader which do the same thing as my `RawFileReader` so let me just use it instead